Poster/index.html

```diff
@@ -5,25 +5,43 @@
     <meta charset="UTF-8">
```

Comment:
On line 2, note the language. This should be updated based on the content of the poster to:
`<html lang="fr">` 👍